milli/src/search/mod.rs

```diff
@@ -105,7 +115,7 @@ impl<'a> Search<'a> {
                 let result = analyzer.analyze(query);
                 let tokens = result.tokens();
                 builder.build(tokens)?
-            },
+            }
```

Comment:
```suggestion
            },
```
We chose to keep the coma on every match cases